WalletWasabi/Tor/Socks5/Pool/TorHttpPool.cs

```diff
@@ -39,7 +39,7 @@ public class TorHttpPool : IDisposable
 {
 	/// <summary>Maximum number of <see cref="TorTcpConnection"/>s per URI host.</summary>
 	/// <remarks>This parameter affects maximum parallelization for given URI host.</remarks>
-	public const int MaxConnectionsPerHost = 100;
+	public const int MaxConnectionsPerHost = 500;
```

Comment:
I was not aware that one can do CJ for many wallets at the same time. 

500 is a value that one should never reach (trademark here) unless one does CJ with like 10 wallets at the same time. It's unclear what happens when we reach that many Tor connections & circuits though because Tor has some limits too. 

For ordinary users, 500 is not reachable but it still should allow us to discover bugs.

Reply:
Ok, I will revert it.